src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriversHierarchyVisitor.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.drivers.visitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverComposite;
+import edu.kis.powp.jobs2d.drivers.adapter.DriverDecorator;
+
+public class DriversHierarchyVisitor implements IVisitor {
+    @Override
+    public void visit(DriverComposite driver) {
+    }
+
+    @Override
+    public void visit(DriverDecorator driver) {
+    }
+
+
+    @Override
+    public void visit(Job2dDriver driver) {
+    }
```

Comment:
Please provide a working example, e.g., you can count drivers which compose the current one.